src/lib.rs

```diff
@@ -174,7 +173,7 @@ impl AstarteSdk {
         Ok(())
     }
 
-    async fn poll_connack(&mut self) -> Result<(), AstarteError> {
+    async fn astarte_device_connack_event(&mut self) -> Result<(), AstarteError> {
```

Comment:
See observations in the other comment for prefix and naming, I'd call this `wait_for_connack`